admin/Routes.js

```diff
@@ -47,9 +46,10 @@ class Routes extends Component {
         const selectedLang = Object.keys(langOptions).filter(
             key => langOptions[key]
         );
-        const locale = language[selectedLang[0]];
+        const lang = selectedLang[0];
+        const i18nConfig = getI18nWithDefaultLang(lang);
```

Comment:
I have wrapped `i18n.js` inside a function so that we can set the user selected language.